src/model.ts

```diff
@@ -114,6 +115,7 @@ export type Mixin<S extends Stage> = BaseNode<'Mixin', S> & {
 
 export type ObjectMember<S extends Stage> = Field<S> | Method<S>
 export type ClassMember<S extends Stage> = Constructor<S> | ObjectMember<S>
+export type DescribeMember<S extends Stage> = Field<S> | Fixture<S> | Test<S> | Method<S>
```

Comment:
If we are not using this anywhere else, we might as well just use the type union on the Describe.members field

Reply:
It appears here too:
https://github.com/uqbar-project/wollok-ts/pull/11/files#diff-a6322b30cb17d35f81779ff7980ea4c3R32